### PR TITLE
add: `lyx`

### PR DESCRIPTION
**Pull request by cat-master21**
Will fail because #970 hasn't been merged yet.

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
```
The following packages have unmet dependencies:
 lyx : Depends: libhunspell-1.6-0 but it is not installable
       Recommends: texlive-latex-recommended but it is not going to be installed
       Recommends: texlive-latex-extra but it is not going to be installed
       Recommends: texlive-science but it is not going to be installed
       Recommends: texlive-generic-recommended but it is not installable
       Recommends: texlive-generic-extra but it is not installable
       Recommends: texlive-fonts-recommended but it is not going to be installed
       Recommends: preview-latex-style but it is not going to be installed
       Recommends: dvipng but it is not going to be installed
       Recommends: imagemagick
       Recommends: psutils
       Recommends: ghostscript
       Recommends: poppler-utils
       Recommends: fonts-lyx but it is not going to be installed
       Recommends: evince-gtk but it is not installable or
                   pdf-viewer
       Recommends: elyxer but it is not installable or
                   tex4ht or
                   hevea but it is not going to be installed or
                   tth but it is not going to be installed or
                   latex2html but it is not going to be installed
[!] ERROR: Failed to install the package
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Errors with
```
The following packages have unmet dependencies:
 lyx : Depends: lyx-common (= 2.3.6.1-1~focal~ppa1) but 2.3.6.1-1~bionic~ppa1 is to be installed
       Recommends: texlive-latex-recommended but it is not going to be installed
       Recommends: texlive-latex-extra but it is not going to be installed
       Recommends: texlive-science but it is not going to be installed
       Recommends: texlive-generic-recommended but it is not installable
       Recommends: texlive-generic-extra but it is not installable
       Recommends: texlive-fonts-recommended but it is not going to be installed
       Recommends: preview-latex-style but it is not going to be installed
       Recommends: dvipng but it is not going to be installed
       Recommends: imagemagick
       Recommends: psutils
       Recommends: ghostscript
       Recommends: poppler-utils
       Recommends: fonts-lyx but it is not going to be installed
       Recommends: evince-gtk but it is not installable or
                   pdf-viewer
       Recommends: elyxer but it is not installable or
                   tex4ht or
                   hevea but it is not going to be installed or
                   tth but it is not going to be installed or
                   latex2html but it is not going to be installed
```

**Comment by saenai255** [comment]
Please follow the common PR format for adding packages. Did you contact the devs?

**Comment by cat-master21** [comment]
> Please follow the common PR format for adding packages. Did you contact the devs?

Done via email: https://www.lyx.org/MailingLists#toc2

File: packages/lyx/lyx.pacscript [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+depends="libqt5svg5-dev texlive-base python3 imagemagick libhunspell-dev file hicolor-icon-theme desktop-file-utils"
+build_depends="bc libx11-dev qtbase5-dev libenchant-2-dev"
+optdepends=("rcs: built-in version control system"
+  "texlive-latex-base: float wrap support")
+url="https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-${version}.tar.gz"
+hash="6d6f5458ebaac644cdfa35114d029e9ec57b4d930268d6d40bd9795d5c7e7929"
+maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
+repology=("project: lyx")
+prepare() {
+true
```
Comment (Henryws) [inline packages/lyx/lyx.pacscript:13]: Tab this

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/lyx/lyx.pacscript [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+version="2.3.6.1"
+description="An advanced WYSIWYM document processor & LaTeX front-end"
+depends="libqt5svg5-dev texlive-base python3 imagemagick libhunspell-dev file hicolor-icon-theme desktop-file-utils"
+build_depends="bc libx11-dev qtbase5-dev libenchant-2-dev"
+optdepends=("rcs: built-in version control system"
+  "texlive-latex-base: float wrap support")
+url="https://ftp.lip6.fr/pub/lyx/stable/2.3.x/lyx-${version}.tar.gz"
+hash="6d6f5458ebaac644cdfa35114d029e9ec57b4d930268d6d40bd9795d5c7e7929"
+maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
+repology=("project: lyx")
```
Comment (Henryws) [inline packages/lyx/lyx.pacscript:11]: Add an line between `repology` and `prepare`

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**